src/components/ui.css.ts

```diff
@@ -18,29 +18,37 @@ export const container = style({
   paddingRight: theme.space[4],
 })
 
+export enum Containers {
+  Normal = "normal",
+  Wide = "wide",
+  Narrow = "narrow",
+  Tight = "tight",
+  Fullbleed = "fullbleed",
+}
```

Comment:
My understanding of how to set this up might be a little off, but for the string prop values, couldn't we have something like `type containerWidth = 'normal' | 'wide' | 'narrow' | 'tight' | 'fullbleed'` (or something like `keyof typeof Containers`) and keep the same API for JS but also support thing like autocomplete?

Reply:
And I'm just seeing this comment now after making my other one above. Yep. You're right. Expect a commit updating all these soon.